### DISCUSSION TITLE
Notifications for Smart Folders

### DISCUSSION THREAD

**Discussion by sherlock440** [ViennaRSS/vienna-rss#1641] [Ideas]
As for me, Smart Folders is the main feature of the Vienna (especially after the moment when the old ReadKit app was gone). But there is one problem: Smart Folder have neither badges of unread notifications nor push-notifications.

In my opinion it is a big omission, because such a powerful RSS app is used for work purposes, where it is needed to monitor news with specific words and other filtering.

I am pretty sure that it will be really useful if Vienna get option to disable typical notifications and enable notifications for Smart Folders (or badges of unread articles at least)

**Comment by barijaona** [comment]
Duplicate of #532 and #1621 

I agree with you but I am still searching a smart implementation.
